README.md

```diff
@@ -1,128 +1,208 @@
-            this.specs = ['tests/desktop'];
+            }
         }
     }
-};
+}
 ```
 
-## Описание конфигурационного файла
+### specs
+**Required.** The list of pathes where `hermione` will look for tests.
+
+For example,
+```javascript
+specs: [
+    'tests/desktop',
+    'tests/touch'
+]
+```
 
-### Настройки браузеров
+### browsers
+**Required.** The list of browsers which should be used for running tests.
 
-Браузеры, в которых необходимо запускать тесты, настраиваются в секции `browsers`.
-Формат секции:
-```js
+Browser section has the following format
+```javascript
 browsers: {
     <browser_id> {
-        <setting>:<value>
-        <setting>:<value>
+        <option>:<value>
+        <option>:<value>
     }
 }
 ```
-Значение `<browser-id>` используется в отчёте для идентификации браузера.
-Доступные настройки браузера:
+`<browser-id>` valus is used for browser identification.
 
-* `desiredCapabilities` (обязательная) - Необходимые для этого браузера WebDriver [DesiredCapabilites](https://github.com/SeleniumHQ/selenium/wiki/DesiredCapabilities)
-* `sessionsPerBrowser` - Количество одновременно запущеных сессий для браузера с данным id. По умолчанию 1.
-* `retry` - Сколько раз тест может быть перезапущен. По умолчанию 0.
+Available browser options:
 
-### Подготовка webdriver-сессии к работе
-Подготовка сессии к работе (например, установка специфических для пользователя команд) выполняется в секции `prepareBrowser`.
-Формат секции:
-```js
-prepareBrowser: function(browser) {
-    // do setup here
-}
-```
+Option name               | Description
+------------------------- | -------------
+`desiredCapabilities`     | **Required.** Used WebDriver [DesiredCapabilites](https://github.com/SeleniumHQ/selenium/wiki/DesiredCapabilities)
+`sessionsPerBrowser`      | Number of sessions which are run simultaneously. Default value is `1`.
+`retry`                   | How many times test should be rerun. Default value is `0`.
 
-В данную функцию будет передана сессия `webdriver.io`.
+### grid
+Selenium grid URL. Default value is `http://localhost:4444/wd/hub`.
 
-### Кастомизация настройки окружения
+### baseUrl
+Base service url. Default value is `http://localhost`.
 
-Данные в конфиге можно изменять в зависимости от дополнительных условий в функции `prepareEnvironment`. Использование этой функции не обязательно, она для удобства.
+### timeout
+Time for text execution. Default value is `60000` ms.
 
-## Плагины
-hermione поддерживает расширения.
-Плагин представляет из себя модуль, экспортирующий одну единственную функцию. Эта функция принимает на вход два параметра:
-* инстанс hermione
-* опции плагина из конфига
+### waitTimeout
+Time for web page events. Default value is `10000` ms.
 
-В момент запуска тестов плагин будет загружен и вызван с текущим инстансом hermione и опциями конфига, соответствующими данному плагину.
+### slow
+If test execution time greater then this value, then test is slow. Default value is `10000`.
```

Comment:
@j0tunn should be described more I guess. I still don't quite understand the usage of this option.